spec/index.md

```diff
@@ -3616,6 +3626,68 @@ S with
 
 ```
 
+
+#### IC Management Canister: Upload Chunk
+
+The controller of a canister, or the canister itself can upload chunks to the chunk store of that canister.
```

Comment:
```suggestion
A controller of a canister, or the canister itself can upload chunks to the chunk store of that canister.
```
Since there might be multiple controllers.